hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/MergeOnReadSnapshotRelation.scala

```diff
@@ -47,9 +50,27 @@ class MergeOnReadSnapshotRelation(sqlContext: SQLContext,
 
   override type FileSplit = HoodieMergeOnReadFileSplit
 
-  override lazy val mandatoryFields: Seq[String] =
+  /**
+   * NOTE: These are the fields that are required to properly fulfil Merge-on-Read (MOR)
```

Comment:
nit: type `fulfil`

Reply:
It's already in that form. Did you mean `fulfill`? Current version is an American version, why w/ `ll` is the UK one

Reply:
yeah, I mean typo.  I don't see `fulfil` that often.